app/src/main/java/com/dimelo/sampleapp/RcConfigurationActivity.java

```diff
@@ -81,6 +117,41 @@ public void update(View v) {
             dimelo.setUserIdentifier(textInputLayout.getEditText().getText().toString());
         }
 
+        if (userNameInputLayout.getEditText().getText().toString() != null && !userNameInputLayout.getEditText().getText().toString().contains(" ")) {
```

Comment:
`toString()` cannot be null so you have to check if `userNameInputLayout != null` && `userNameInputLayout.getEditText() != null`

Reply:
yes i can check this like that !textInputLayout.getEditText().getText().toString().contains(" ")

Reply:
7267aad6bf0ca71a050cbd6ae37249737f3357e5